HentaiWorld/dist/metadata.json

```diff
@@ -12,14 +12,15 @@
   "url": "www.hentaiworld.eu",
   "version": "1.0",
```

Comment:
Update version

Reply:
Did i change it? i don't know how to use github... sorry, i think i changed